mmcv/runner/hooks/logger/text.py

```diff
@@ -14,6 +15,12 @@ def log(self, runner):
             log_str = 'Epoch({}) [{}][{}]\t'.format(runner.mode, runner.epoch,
                                                     runner.inner_iter + 1)
         if 'time' in runner.log_buffer.output:
+            tot_time_sec = runner.log_buffer.output['time'] + \
+                runner.log_buffer.output['data_time']
+            eta_sec = tot_time_sec * \
+                (len(runner.data_loader) * runner.max_epochs - runner._iter)
+            eta_str = str(datetime.timedelta(seconds=int(eta_sec)))
+            log_str += ('eta: {}, '.format(eta_str))
             log_str += (
                 'time: {log[time]:.3f}, data_time: {log[data_time]:.3f}, '.
                 format(log=runner.log_buffer.output))
```

Comment:
Would it be better to show time and data_time first and then eta?

Reply:
In my opinion (as a user), I prefer to see eta first. Time and data_time is important for performance comparison but they are nearly unchanged during training so less informative from a user point of view.

Reply:
This also makes sense, the order may be left unchanged.